monitoring/types.go

```diff
@@ -7,13 +7,18 @@ import (
 const (
 	longhornName = "longhorn"
 
-	subsystemVolume      = "volume"
-	subsystemBackupStats = "backup_stats"
-	subsystemNodeStats   = "node_stats"
+	subsystemVolume          = "volume"
+	subsystemBackupStats     = "backup_stats"
+	subsystemNodeStats       = "node_stats"
+	subsystemInstanceManager = "instance_manager"
+	subsystemManager         = "manager"
```

Comment:
These can then be remove and instead use NodeLabel, InstanceManagerLabel, etc from below.

Reply:
I like to keep the `subsystem*` prefix for those labels because it makes the naming convention clear. e.g:
```
prometheus.BuildFQName(longhornName, InstanceManagerLabel, "cpu_usage_millicpu")
```
How do you think?

Reply:
Fair enough, how about this instead?
instead of using `NodeLabel` and `subsystemNode` just use one field called `NodeComponent` for both?
Since you are effectively tracking metrics for a node, volume, manager, instance_manager.

If you prefer though you can also keep the subsystem but then consider putting the assignment on the same line as the NodeLabel
subsystemNode = nodeLabel = "node" if possible?

I have no hard feelings about this, as long as the actual value is the same as the individual component labels.